app/views/layouts/_rails_default.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %>
```

Comment:
không bị quá 80 ký tự thì không nên xuống dòng như này
thay toàn bộ ' ' bằng " "
thêm dòng trắng ở cuối